devimint/src/main.rs

```diff
@@ -877,8 +877,14 @@ async fn do_try_create_and_pay_invoice(
     // Verify that after the lightning node has restarted, the gateway
     // automatically reconnects and can query the lightning node
     // info again.
-    let mut info_cmd = cmd!(gw, "info");
-    assert!(info_cmd.run().await.is_ok());
+    poll("Waiting for info to succeed after restart", || async {
+        let mut info_cmd = cmd!(gw, "info");
+        Ok(info_cmd
+            .run()
+            .await
+            .map_err(|e| warn!("Info command not ready yet, retrying: {e}"))
+            .is_ok())
+    });
```

Comment:
Not sure if re-trying is the right thing to do, could also make it an expect for further investigation.